imports/client/components/SetupPage.tsx

```diff
@@ -4,7 +4,6 @@ import { Meteor } from 'meteor/meteor';
 import { OAuth } from 'meteor/oauth';
 import { useSubscribe, useTracker } from 'meteor/react-meteor-data';
 import { ServiceConfiguration } from 'meteor/service-configuration';
-import { _ } from 'meteor/underscore';
```

Comment:
Heh, looks like that's been unused since da6579de9cfa172308bde273bdf9f7c6519d75c6.  Do we have systematic linting for unused imports now, or was this something that VSCode flagged for you that you happened to clean up while you were here?

Reply:
This got picked up my `@typescript-eslint/no-unused-vars`, which is just a version of the standard `no-unused-vars` that works better with TS.